openshift_tools/ircbot/requirements.txt

```diff
@@ -1 +1,2 @@
 sopel
+git+https://github.com/nithinmurali/pygsheets.git#staging
```

Comment:
why not just `pygsheets`?

Reply:
Without my merged PR (https://github.com/nithinmurali/pygsheets/pull/147)...it spams stdout with useless messages. But it's not a necessity...just annoying =P